build.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'com.adarshr.test-logger' version '1.6.0'
```

Comment:
Why did you choose this logger?

Reply:
it is logger for the test output

Reply:
Yes, I know. But why don't you use regular logger?

Reply:
It just looks better https://github.com/radarsh/gradle-test-logger-plugin